### ISSUE TITLE
[Feature Request] Copy link options 

### ISSUE THREAD

**Issue by bratan997** [insin/control-panel-for-twitter#641]
![image](https://github.com/user-attachments/assets/54a52633-c0d3-4d76-9e95-1cb61fe41428)
Add the copy link options such as fixupx, fxtwitter, or vxtwitter. The examples are from Old Twitter Layout (2024) extension.

**Comment by kinsi55** [comment]
Yes, automatically having the link on the tweet-date be a selected option would be huge
